apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,195 @@
+const Big = require('big.js')
+const dayjs = require('dayjs')
+const { get } = require('lodash')
+
+const { FinanceInfoClient } = require('@open-condo/clients/finance-info-client/FinanceInfoClient')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getByCondition } = require('@open-condo/keystone/schema')
+
+const { buildExportFile, DOCX_FILE_META } = require('@condo/domains/common/utils/createExportFile')
+const { renderMoney } = require('@condo/domains/common/utils/money')
+const { moneyToWords } = require('@condo/domains/common/utils/moneyToWords')
+const { buildUploadInputFrom } = require('@condo/domains/common/utils/serverSchema/export')
+const { normalizeTimeZone } = require('@condo/domains/common/utils/timezone')
+const { DEFAULT_INVOICE_CURRENCY_CODE, INVOICE_STATUS_CANCELED } = require('@condo/domains/marketplace/constants')
+const { Invoice } = require('@condo/domains/marketplace/utils/serverSchema')
+const { DEFAULT_ORGANIZATION_TIMEZONE } = require('@condo/domains/organization/constants/common')
+const { TICKET_DOCUMENT_GENERATION_TASK_FORMAT } = require('@condo/domains/ticket/constants/ticketDocument')
+const { formatDateToTimezone } = require('@condo/domains/ticket/utils')
+
+const logger = getLogger('generateDocumentOfPaidWorksCompletion')
+
+const financeInfoClient = new FinanceInfoClient()
+
+async function getLocaleToPullFinanceInfoClient (locale, organization) {
+    let psrn = null, organizationAddress = null, iec = null
+
+    if (locale === 'ru') {
+        try {
+            ({ iec, psrn, organizationAddress } = await financeInfoClient.getOrganization(organization.tin))
+        } catch (error) {
+            logger.info({ msg: 'fall financeInfoClient when get organization by tin', organizationId: organization.id, tin: organization.tin, error: error })
+        }
+    }
+
+    return { iec, psrn, organizationAddress }
+}
+
+const changeUndefinedDataToBlanks = (documentDataInLine) => {
```

Comment:
What is documentDataInLine ? What type is this variable?

Why not simply documentData?